src/components/structures/MatrixChat.js

```diff
@@ -674,6 +674,7 @@ export default createReactClass({
                     this.state.view !== VIEWS.E2E_SETUP
                 ) {
                     this._onLoggedIn();
+                    ThemeController.isLogin = false;
```

Comment:
Hmm, would it be better to set this inside `_onLoggedIn`, since there are other paths that call it as well?

Reply:
good call